### ISSUE TITLE
[Regression] Extraneous vertical scrollbar since version 0.75.0

### ISSUE THREAD

**Issue by anisabboud** [arnog/mathlive#1521]
#### 1. Extraneous scrollbar
- Up to version 0.74.0, mathlive wasn't showing an inner scrollbar (not needed).
  ![image](https://user-images.githubusercontent.com/5672108/176432004-f5799841-abf6-47b7-96d7-0d44d59627fa.png)
- Starting in version 0.75.0 (and 0.76.0), mathlive shows extraneous inner scrollbar.
  This happens on the official demo as well https://cortexjs.io/mathlive/:
  ![image](https://user-images.githubusercontent.com/5672108/176432588-924242a9-8563-439b-8df2-f85683519bbd.png)

#### 2. Vertical alignment
- Prior to version 0.75.0, using `display: inline-block` on the math-field was working well and the field was centered vertically automatically next to other text:
  ![image](https://user-images.githubusercontent.com/5672108/176431846-3fd5010b-a89d-4ce7-a7cc-c368a659cb4a.png)
- Starting in version 0.75.0, the math-field jumps high up when placed next to other text:
  ![image](https://user-images.githubusercontent.com/5672108/176433482-65ada042-9156-4b82-930c-3fefb10d7a09.png)

### Workaround
Both issues can be worked around by adding CSS:
```css
math-field {
  overflow: unset;
}
```

**Comment by sbondaryev** [comment]
Caused by the class `.ML__sr-only` (`height: 1px`)
Step to reproduce:
- Go to the demo page https://cortexjs.io/mathlive/demo/
- Set `padding:0px` for the  tag `math-field`
![scroll](https://user-images.githubusercontent.com/2843147/176439567-0ed80ccd-7da9-47e9-937b-cfd83a93e5d9.gif)

**Comment by sbondaryev** [comment]
@anisabboud  I think that the `overflow: unset;` will disable the feature added in 0.75.0 
#970 It is now possible to vertically scroll the content of the mahtfield. (see [here](https://github.com/arnog/mathlive/blob/master/CHANGELOG.md#0750-2022-06-21))
You can try 
```
math-field {
  padding: 1px;
}
```

**Comment by arnog** [comment]
Good catch. Thanks for reporting this. I have published `mathlive@0.76.1` which includes a fix.

**Comment by arnog** [comment]
Another note regarding the second issue about vertical-alignment.

This is, sadly, browser engine specific. 

WebKit will continue to align the baselines, while Blink won't. Even more sadly, they are both valid interpretation of the CSS spec. The behavior is defined in the [CSS Visual Formatting Model](https://www.w3.org/TR/CSS2/visudet.html#leading):
> The baseline of an 'inline-block' is the baseline of its last line box in the normal flow, unless it has either no in-flow line boxes or if its 'overflow' property has a computed value other than 'visible', in which case the baseline is the bottom margin edge.

The reasoning behind this is that since the inline-block *could* overflow, it might be  too onerous to determine its baseline (since some of its content may not be visible/rendered). However, one could argue that if it *doesn't* overflow, the baseline is known and it should continue to align with the baseline. This is what the WebKit team has implemented and [argued with the W3C CSSWG, which agreed](https://lists.w3.org/Archives/Public/www-style/2015Mar/0349.html). So now, we have two incompatible implementations...

The way to resolve this is either to set unset the overflow property on the `<math-field>` element, as you suggested, or, what I would recommend, is to set the layout of the parent container to `display:flex; align-items: baseline;`.

**Comment by anisabboud** [comment]
Thank you @sbondaryev and @arnog!

I can confirm that `0.76.1` indeed fixes the scrollbar issue. Thanks!
Regarding the vertical alignment issue, I worked around it using:
```css
math-field {
  vertical-align: middle;
}
```
(This seems to work fine on Chrome for the simple use-case. Haven't tried it on WebKit / complex use-cases.)

**Comment by arnog** [comment]
@anisabboud that can work in some cases, but note that you're aligning on the middle, not the baseline, so for example `\frac{\frac12}3` will appear mis-aligned.
